libcxx/test/std/strings/string.view/string.view.ops/subview.pass.cpp

```diff
@@ -0,0 +1,20 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: std-at-least-c++26
+
+// <string_view>
+
+// constexpr basic_string_view subview(size_type pos = 0,
+//                                     size_type n = npos) const;      // freestanding-deleted
+
+int main(int, char**) {
+  // This test is intentionally empty because subview is an alias for substr
+  // and is tested in substr.pass.cpp.
```

Comment:
Looks like we don't need this file.

Reply:
I am not sure what the current practice about function aliases is. Normally we want a test file per function. This one is to tell the user what to look at. Are there any similar cases in the library?

Reply:
WDYT: Should I remove the file now? How we are supposed to tell the user where to look after the actual implementation of the test otherwise?
Maybe rename the other test file to `substr.subview.pass.cpp`? Would that be OK?

Reply:
Personally, I'd either keep this file as-is or rename the other test file to `substr.subview.pass.cpp`. I wouldn't just remove this file, since we'll end up searching for the tests and not finding them.